include/ipmitool/ipmi_fru.h

```diff
@@ -598,21 +598,34 @@ struct fru_picmgext_amc_link_desc_record {
 /* FRU Board manufacturing date */
 #define FRU_BOARD_DATE_UNSPEC 0 /* IPMI FRU Information Storage Definition
                                    v1.0 rev 1.3, Table 11-1 */
+
+static inline uint32_t SECS_FROM_1970_1996=820454400;
```

Comment:
Why did you demote it to `uint32_t` while moving it to global namespace?
Actually, I think it would be the best to use `time_t`. And it has to be `const`, on the contrary, `inline` doesn't seem to do anything here.